src/modules/group-changes.js

```diff
@@ -10,29 +10,37 @@ const _groupChanges = (diff, oldDefinitions, newDefinitions) => {
       const { location } = change;
       const { path } = location;
 
-      const drdElementType = _getDrdElementType(path);
-      const drdElementPath = _extractPathUntilTarget(path, drdElementType);
-      const drdElementId = _getDrdElementId(
-        changeType === "removed" ? oldDefinitions : newDefinitions,
-        drdElementPath
-      );
-
-      if (drdElementId) {
-        // Set changeType
-        _setDrdElementChangeType(
-          grouped,
-          oldDefinitions,
-          newDefinitions,
-          drdElementPath,
-          drdElementId
+      const isRootLevelAttributeChange = path.split(".").length === 1;
```

Comment:
```suggestion
      const isRootLevelAttributeChange = path.split(".").length === 1;
```

Does it make a difference if this is an attribute or general root level change? If not, let's simplify it:

```
if (isRootLevelChange(path)) { 

} else {
  ...
}
```

Reply:
✅